### ISSUE TITLE
cargo build fails

### ISSUE THREAD

**Issue by kayvank** [developeruche/chronicle#4]
cargo build fails with this error:

```
$ cargo build --release
error: failed to load manifest for workspace member `/home/kayvan/dev/workspaces/workspace-q2io/rust/wt/chronicle/nixify/bin/chronicle`
referenced by workspace at `/home/kayvan/dev/workspaces/workspace-q2io/rust/wt/chronicle/nixify/Cargo.toml`

Caused by:
  failed to parse manifest at `/home/kayvan/dev/workspaces/workspace-q2io/rust/wt/chronicle/nixify/bin/chronicle/Cargo.toml`

Caused by:
  dependency (tokio) specified without providing a local path, Git repository, version, or workspace dependency to use
```

**Comment by kayvank** [comment]
The issue is related to: [workspace = true](https://github.com/developeruche/chronicle/blob/main/Cargo.toml#L17)
Changing it to:
```
tokio = { version = "1.37", features = ["macros", "rt-multi-thread", "signal", "sync"] }
```
fixes the issue

**Comment by developeruche** [comment]
Ooops... apologies!

Make a PR if you can to implement this fix

**Comment by kayvank** [comment]
I would like to open an issue to nixify the project. Are you open to that? If so, I can do a PR for that work as well.

**Comment by developeruche** [comment]
> nixify

Sure! Go for it

**Comment by kayvank** [comment]
Thank you, created #7

**Comment by developeruche** [comment]
Closed by #5
